main.go

```diff
@@ -93,15 +110,18 @@ func getConfig() (*config, error) {
 		return &config{}, fmt.Errorf("set GITLAB_BASE_URL")
 	case integrationDirectory == "":
 		return &config{}, fmt.Errorf("set INTEGRATION_DIRECTORY")
+	case osToEntGithubToken == "":
+		return &config{}, fmt.Errorf("set OS_SYNC_GITHUB_SECRET")
 	}
 
 	return &config{
-		githubSecret:          []byte(githubSecret),
-		githubToken:           githubToken,
-		gitlabToken:           gitlabToken,
-		gitlabBaseURL:         gitlabBaseURL,
-		watchRepositories:     repositoryWatchList,
-		integrationDirectory:  integrationDirectory,
+		githubSecret:         []byte(githubSecret),
+		githubToken:          githubToken,
+		gitlabToken:          gitlabToken,
+		gitlabBaseURL:        gitlabBaseURL,
+		watchRepositories:    repositoryWatchList,
+		integrationDirectory: integrationDirectory,
+		osToEntGithubToken: osToEntGithubToken,
```

Comment:
Format: missing space.

Reply:
Good eye. In general, the go-1.13 formatter does a lot of changes here if I run it. Should I add a separate commit with the formatting run, or keep it on the old style you think?

Reply:
IMO, no need for a different commit because it is only this part.

Reply:
New style :+1: